audio_play/src/audio_play.cpp

```diff
@@ -109,6 +119,16 @@ namespace audio_transport
         g_object_unref (audiopad);
       }
 
+     static void cb_need_data (GstElement *appsrc,
+                   guint       unused_size,
+                   gpointer    user_data)
+     {
+       ROS_DEBUG("need-data signal emitted! Pausing the pipeline");
```

Comment:
This should probably be a warning message, so that the user knows that something is wrong with the audio stream.